transmart-core-db/src/integration-test/groovy/org/transmartproject/db/multidimquery/QueryServicePgSpec.groovy

```diff
@@ -463,13 +463,9 @@ class QueryServicePgSpec extends Specification {
             dimension.asSerializable(it)
         }
 
-        then: "List of all trial visits matching the constraints is returned"
+        then: "List of all visits matching the constraints is returned"
         visits.size() == expectedResult.size()
-        expectedResult.every { expect ->
-            visits.any {
-                [expect.id] == [it.id]
-            }
-        }
+        visits.collect { it.encounterIds['VISIT_ID'] }.sort() == expectedResult.collect { it.encounterIds['VISIT_ID']}.sort()
 
         //TODO Fix counting elements of visit dimension - find a way to count distinct on two properties
```

Comment:
isn't this one fixed now?

Reply:
I should indeed have been fixed now, having a single identifier. But it seems that `getDimensionElementsCount` does not exist anymore. I will remove these lines from the test.